src/beam_stats.app.src

```diff
@@ -12,8 +12,18 @@
   {mod, { beam_stats_app, []}},
 
   {env,
-    [ {production_interval , 30000}
-    , {consumers, []}
+    [ {production_interval , 5000}
+    , {consumers, [
+        {beam_stats_consumer_graphite,
+        [ {consumption_interval , 5000}
+        , {host                 , "localhost"}
+        , {port                 , 2003}
+        , {timeout              , 5000}
+        ]} 
+    , {beam_stats_consumer_csv,
+      [ {consumption_interval , 5000}
+      , {path                 , "beam_stats.csv"}
+      ]}    
+    ]}
```

Comment:
I intentionally did not include these in the default config, since nothing should be started unless _explicitly_ configured to.